tyanhan/markbind
# Fix untidy tooltips and popovers

**What is the purpose of this pull request?**




**Overview of changes:**
Fixes [#1969](https://github.com/MarkBind/markbind/issues/1969)
![image](https://user-images.githubusercontent.com/68331979/182029158-e2df88c3-5f4a-4c07-ace0-566a876abc8e.png)
![image](https://user-images.githubusercontent.com/68331979/182029170-f5878717-12bf-48da-aea7-c237c5638b9c.png)


**Anything you'd like to highlight/discuss:**
Modified markbind.css

**Testing instructions:**
1. Create test site
2. Add a tooltip or popover component to any page
3. Run markbind serve -d and verify that component is correct



**Proposed commit message: (wrap lines at 72 characters)**
Removed unintended borders around tooltips and popovers by editing the css



---

**Checklist:** :ballot_box_with_check:
